exporters/otlp/common/build.gradle.kts

```diff
@@ -9,7 +9,7 @@ plugins {
 }
 
 description = "OpenTelemetry Protocol Exporter"
-otelJava.moduleName.set("io.opentelemetry.exporter.otlp.internal")
+otelJava.moduleName.set("io.opentelemetry.exporter.internal.otlp")
```

Comment:
what affect does this have?

Reply:
The `moduleName` is used to [set the location](https://github.com/open-telemetry/opentelemetry-java/blob/main/buildSrc/src/main/kotlin/otel.java-conventions.gradle.kts#L145-L158) of the `version.properties` file. It should reflect the base package of the artifact, so in this case it was wrong. If its not correct, then fetching a relative resource like `OtlpUserAgent.class.getResourceAsStream("version.properties")` will fail.